compiler/lib/src/main/scala/codegen/ComputeGeneratedFiles.scala

```diff
@@ -15,13 +15,13 @@ object ComputeGeneratedFiles {
     } 
     yield xmlFiles ++ cppFiles
 
-  def getCppFiles(tul: List[Ast.TransUnit]) =
+  def getCppFiles(tul: List[Ast.TransUnit]): Either[Error,List[String]] =
```

Comment:
Can we use Result.Result instead of Either[...] here? Similarly for the other uses of Either.

Reply:
Done